### ISSUE TITLE
Feature request: Ambience restricted to certain rooms/areas

### ISSUE THREAD

**Issue by Pia86** [LostArtefacts/TRX#811]
Ability to have ambience restricted to certain rooms/areas. Like in TR3 or TR4 where it’s possible to have the ambience change mid-level. I’d like to have that opportunity in custom levels.

**Comment by mizunosuisei** [comment]
Hi! Apologies if I've misinterpreted your issue, but if it's regarding audio ambience (such as windy outdoors, and cave indoors music all in the same level) i believe it involves trigger bits for the classic tr games
https://www.tombraiderforums.com/showthread.php?t=79194

**Comment by Pia86** [comment]
Yeah, that is what I mean. It's not possible in TR1 and TR2, as far as I know, though.

**Comment by rr-** [comment]
This is not going to happen within TR1 as it'd require changing the levels format. Why not use TR3+?

**Comment by Pia86** [comment]
I just thought it would be neat to have the option for TR1 as well. But if it can't be done, I'll make do.
